### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by rbn30117**
Add option to auto publish with a cron service and deploy hooks.

### CODE REVIEW

**Review by stullesha — COMMENTED** [review]
change cron job link

File: content/en/troubleshooting/faq.md [hunk 1]
```diff
@@ -35,7 +35,9 @@ Yes you can! See [Configure with Environment Variables](/getting-started/configu
 How to automate the "publish at intervals" part depends on your situation:
 
 * If you deploy from your own PC/server, you can automate with [Cron](https://en.wikipedia.org/wiki/Cron) or similar.
-* If your site is hosted on a service similar to [Netlify](https://www.netlify.com/) you can use a service such as [ifttt](https://ifttt.com/date_and_time) to schedule the updates.
+* If your site is hosted on a service similar to [Netlify](https://www.netlify.com/) you can--
```
Comment (jmooring) [inline content/en/troubleshooting/faq.md:38]:
```suggestion
* If your site is hosted on a service similar to [Netlify](https://www.netlify.com/) you can:
```

File: content/en/troubleshooting/faq.md [hunk 2]
```diff
@@ -35,7 +35,9 @@ Yes you can! See [Configure with Environment Variables](/getting-started/configu
...
+  * use a service such as [ifttt](https://ifttt.com/date_and_time) to schedule the updates.
```
Comment (jmooring) [inline content/en/troubleshooting/faq.md:39]:
```suggestion
  * Use a service such as [ifttt](https://ifttt.com/date_and_time) to schedule the updates
```

File: content/en/troubleshooting/faq.md [hunk 3]
```diff
@@ -35,7 +35,9 @@ Yes you can! See [Configure with Environment Variables](/getting-started/configu
...
+  * set up a deploy hook which you can run with a cron service to deploy your site at intervals, such as [cron-job.org](https://cron-job.org/). (Netlify and Cloudflare Pages support deploy hooks).
```
Comment (jmooring) [inline content/en/troubleshooting/faq.md:40]:
```suggestion
  * Set up a deploy hook which you can run with a cron service to deploy your site at intervals, such as [cron-job.org](https://cron-job.org/) (both Netlify and Cloudflare Pages support deploy hooks)
```

**Review by jmooring — CHANGES_REQUESTED** [review]

**Pull request merged**